addOns/graphql/src/main/javahelp/org/zaproxy/addon/graphql/resources/help/contents/graphql.html

```diff
@@ -56,6 +56,13 @@ <h3> Actions </h3>
 
 The definitions will be imported synchronously and any warnings will be returned.
 
+<h3>GraphiQL IDE</h3>
```

Comment:
Add a line or two explaining how this is of benefit to the user?

Blah blah blah, generated traffic is proxied and populates ZAP's Sites Tree yaadaa yaadaa yaadaa